client/account-settings/index.js

```diff
@@ -44,6 +44,6 @@ export default ( { formData, formMeta } ) => ( {
 	},
 
 	View: () => (
-		<AccountSettingsRootView />
+		<AccountSettingsRootView storeOptions={ storeOptions } />
```

Comment:
Don't pass any props to the component, they won't be there in Calypso. Instead, store the `storeOptions` in the Redux state.

Also, this "app" must work even if it doesn't receive any arguments (`{ formData, formMeta, storeOptions }` can be `undefined`), so you'll need to:
* Add `storeOptions` property to the output of the `/wc/v1/connect/account/settings` endpoint. See the `fetchSettings` function on `client/account-settings/state/actions.js`, and the `classes\class-wc-rest-connect-account-settings-controller.php` file. You'll also need to add the `paperSize` there. Also, on the `output_account_screen` function of `classes/class-wc-connect-settings-pages.php`.
* Make the whole UI work when there's no data. See the "isLoading / renderSpinner" logic in the (now deleted) `PaymentMethodSelector` component. A good way to "simulate" it is to remove the 3rd param of the `do_action( 'enqueue_wc_connect_script' ...)` in the `output_account_screen` function of `classes/class-wc-connect-settings-pages.php`.